qtpy/QtLocation.py

```diff
@@ -8,18 +8,27 @@
 """Provides QtLocation classes and functions."""
 
 from . import (
+    parse,
     PYQT5,
     PYQT6,
     PYSIDE2,
     PYSIDE6,
+    PYQT_VERSION,
+    PYSIDE_VERSION,
     QtBindingMissingModuleError,
 )
 
 if PYQT5:
     from PyQt5.QtLocation import *
 elif PYQT6:
-    raise QtBindingMissingModuleError(name='QtLocation')
+    if parse(PYQT_VERSION) >= parse('6.5'):
+        from PyQt6.QtLocation import *
+    else:
+        raise QtBindingMissingModuleError(name='QtLocation')
```

Comment:
If this module was added in Qt 6.5 instead of just in both Python bindings at that same version, then I'm not sure this is the right error, as it is intended for reporting when a particular _binding_ doesn't support a missing module, rather than it not being implemented in Qt itself—not sure how it got overlooked initially, as the current call signature will actually raise an error since it expects `binding` to be passed. Instead, it would seem to be better to use `QtModuleNotInQtVersionError` here, with a tweak to either output the full major + minor version automatically (probably best), or pass that version manually.

Reply:
> If this module was added in Qt 6.5 instead of just in both Python bindings at that same version

This is one of the reasons I need to set up a local environment matrix. I *believe* that PyQt6 had some of these modules the entire time, and it's only PySide6 that just added them to the distribution, but I won't know for sure until I'm able to step through the versions.